### PR TITLE
Bump jetty-setuid to 2.0.1

### LINKED ISSUE #10279: Update Jetty lib-setuid for Jetty 12
**Jetty version(s)**
12.0.0

**Java version/vendor** `(use: java -version)`
17

**OS type/version**
Linux

**Description**
The lib-setuid 1.0.4 is incompatible with Jetty 12.

We need to update lib-setuid to support Jetty 12.

### PR DESCRIPTION
**Pull request by joakime**
Replace `jetty-setuid-java` and the various `jetty-setuid-native` artifacts with the new `jetty-setuid-jna` artifact.

### CODE REVIEW

File: jetty-home/pom.xml [hunk 1]
```diff
@@ -95,30 +95,12 @@
               <artifactItems>
                 <artifactItem>
                   <groupId>org.eclipse.jetty.toolchain.setuid</groupId>
-                  <artifactId>jetty-setuid-java</artifactId>
+                  <artifactId>jetty-setuid-jna</artifactId>
```
Comment (lorban) [inline jetty-home/pom.xml]: Could you please confirm that `jna-5.13.0.jar` will also be copied to `${assembly-directory}/lib/setuid`?

File: jetty-home/src/main/resources/modules/setuid.mod [hunk 2]
```diff
@@ -9,7 +9,7 @@ changing to a restricted user (e.g. Jetty).
 server
 
 [lib]
-lib/setuid/jetty-setuid-java-1.0.4.jar
+lib/setuid/jetty-setuid-jna-2.0.0.jar
```
Comment (lorban) [inline jetty-home/src/main/resources/modules/setuid.mod]: Could you please confirm that adding a `lib/setuid/jna-5.13.0.jar` line isn't needed?

**Review by lorban — CHANGES_REQUESTED** [review]
LGTM, but I have two questions I'd like to be reassured about.

**Comment by joakime** [comment]
This is not ready to review (yet)

I'd like to get the PR at https://github.com/eclipse/jetty.toolchain/pull/281 merged and a new jetty-setuid-jna 2.0.1 released to make integrating easier on this side.

**Comment by lorban** [comment]
I approved https://github.com/eclipse/jetty.toolchain/pull/281, please go on with this PR.

**Review by lorban — APPROVED** [review]
LGTM

**Pull request merged**